examples/basic/apps/web/.eslintrc.js

```diff
@@ -1,3 +1,9 @@
+/** @type {import("eslint").Linter.Config} */
 module.exports = {
-  extends: ["custom/next"],
+  root: true,
+  extends: [require.resolve("@repo/eslint-config/next.js")],
```

Comment:
`require.resolve` shouldn't be needed in these, but are you shooting for less magic?

Reply:
Good call.